app/src/main/java/com/duckduckgo/app/browser/omnibar/KeyboardAwareEditText.kt

```diff
@@ -33,10 +35,25 @@ class KeyboardAwareEditText : AppCompatEditText {
     constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
     constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)
 
+    private var didFocusAlready = false
+
+    private fun Editable.isWebUrl(): Boolean {
+        return Patterns.WEB_URL.matcher(this.toString()).matches()
+    }
+
     override fun onFocusChanged(focused: Boolean, direction: Int, previouslyFocusedRect: Rect?) {
         super.onFocusChanged(focused, direction, previouslyFocusedRect)
+        // stop the auto-selection of text after the first focus
```

Comment:
we don't usually leave comments unless they are really necessary. seems like the code is pretty self-explanatory, so I'd say it's safe to remove them

Reply:
Makes sense, will do